samtranslator/schema/aws_serverless_statemachine.py

```diff
@@ -77,12 +90,12 @@ class CloudWatchEvent(BaseModel):
 
 
 class EventBridgeRuleEventProperties(BaseModel):
-    DeadLetterConfig: Optional[DeadLetterConfig]
-    EventBusName: Optional[PassThrough]
-    Input: Optional[PassThrough]
-    InputPath: Optional[PassThrough]
-    Pattern: Optional[PassThrough]
-    RetryPolicy: Optional[PassThrough]
+    DeadLetterConfig: Optional[DeadLetterConfig] = eventbridgeruleeventproperties("DeadLetterConfig")
+    EventBusName: Optional[PassThrough] = eventbridgeruleeventproperties("EventBusName")
+    Input: Optional[PassThrough] = eventbridgeruleeventproperties("Input")
+    InputPath: Optional[PassThrough] = eventbridgeruleeventproperties("InputPath")
+    Pattern: PassThrough = eventbridgeruleeventproperties("Pattern")
```

Comment:
Why was this changed from `Optional[PassThrough]` to `PassThrough`?

Reply:
Oh it's supposed to be required: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-statemachine-statemachineeventbridgerule.html#sam-statemachine-statemachineeventbridgerule-pattern

Great catch! 👍